invite/views.py

```diff
@@ -35,11 +35,18 @@ def reset(request):
                 password=form.cleaned_data['password'],
             )
             if user is not None:
-                if user.is_active:
-                    login(request, user)
-                    # Redirect to main edit dashboard
-                    return HttpResponseRedirect(
-                        app_settings.INVITE_SIGNUP_SUCCESS_URL)
+                login(request, user)
+                # Redirect to main edit dashboard
+                return HttpResponseRedirect(
+                    app_settings.INVITE_SIGNUP_SUCCESS_URL)
+            else:
+                return render(
+                    request,
+                    'invite/contact.html',
+                    {
+                        'supervisor':  app_settings.INVITE_CONTACT_EMAIL,
+                    }
+                )
```

Comment:
I see the issue #49 points to here, but I don't think this is addressing the problem wholly. While we should check if the user is active before letting them reset the password after getting to the reset form. We should probably check if they are active in [amnesia](https://github.com/unt-libraries/django-invite/blob/24f0fc3755688e728425a95e4307cde85c3be482/invite/views.py#L98) also before sending them a reset link email. From what I can tell, right now, the inactive user will click they forgot their password, enter their email address, get an email with a reset link, fill out the new password, then find out they are inactive.

Reply:
Hmm...checking if the user is active when they enter the email id to reset pswd and then redirect them appropriately is the right way to do it.